src/Models/LoginSession.php

```diff
@@ -2,18 +2,19 @@
 
 namespace SilverStripe\SessionManager\Models;
 
+use UAParser\Parser;
+use InvalidArgumentException;
+use SilverStripe\ORM\DataList;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Control\Session;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
 use SilverStripe\Control\Controller;
 use SilverStripe\Control\HTTPRequest;
-use SilverStripe\Control\Session;
 use SilverStripe\Core\Injector\Injector;
-use SilverStripe\ORM\DataList;
-use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\FieldType\DBDatetime;
-use SilverStripe\Security\Member;
 use SilverStripe\Security\RememberLoginHash;
-use SilverStripe\Security\Security;
 use SilverStripe\SessionManager\Security\LogInAuthenticationHandler;
-use UAParser\Parser;
```

Comment:
Can you please refrain from changes like this in these PRs?
We don't have any specific standards for the order of use statements, so there's no objective improvement here and it's adding clutter and additional mental overhead to reviewing the PR as I now need to check that these are just being moved and not added/removed.

I won't ask you to revert this change now that it's here, but just a request for future PRs.